lib/msal-common/src/url/IUri.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * @hidden
+ */
+export interface IUri {
+    Protocol: string;
+    HostNameAndPort: string;
```

Comment:
Is there any value in separating the hostname and port fields? is there ever a case where we'll have to parse this to only get one or the other?

Reply:
I'm not sure, I believe we do use it when checking for secure uris, but beyond that it is just to make sure the object completely describes the segments of a URI.